lib/main_view_model.dart

```diff
@@ -222,7 +254,7 @@ class MainViewModel with ChangeNotifier {
       customHeaders: {'X-Header-1': 'Value1', 'X-Header-2': 'Value2'},
     );
     observeCurrentCall();
-    _currentCall?.startDebugStats();
+    //_currentCall?.startDebugStats();
```

Comment:
Let's uncomment this, you can just send false for the debug flag in the login method now once mine is merged